package.json

```diff
@@ -22,6 +22,7 @@
     "sjcl": "^1.0.8"
   },
   "devDependencies": {
+    "@types/chrome": "^0.0.174",
```

Comment:
Can we remove this until the fast follow issue, I can't see this being used.

Reply:
done